plugins/inputs/opcua/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Input Plugin: opcua_client
+
+The opcua_client plugin retrieves data from OPCUA slave devices
+
+### Configuration:
+
+```toml
+
+# ## Connection Configuration
```

Comment:
Clean up the comments in the example config here and in sampleConfig.  They need to match the commenting and indentation style because they are used when generating a config.  See https://github.com/influxdata/telegraf/wiki/SampleConfig

Accurate default values should be included here and commented out with a single hash.  Documentation is commented out with two hashes, not hash space hash hash